src/css/components/files/_c-cart-body.scss

```diff
@@ -0,0 +1,7 @@
+.c-cart-body {
+  @include border_style(0,0px, $border_color_dx);
+  display: block;
```

Comment:
`display:block` and `width:100%' not needed, since `div`s have these by default.